bsp/stm32/stm32mp157a-st-ev1/tools/rt-thread-shell.py

```diff
@@ -0,0 +1,26 @@
+import os
```

Comment:
What's meaning of this file?

Reply:
Automate using OpenAMP on the linux side, RT-Thread Shell is easy to use. Does this file only need to be provided to ST?